core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -270,6 +270,8 @@ protected void startJvmMonitorProcess() {
    */
   protected void startServing(String startMessage, String stopMessage) {
     MetricsSystem.startSinks(ServerConfiguration.get(PropertyKey.METRICS_CONF_FILE));
+    LOG.info("Alluxio master web server version {} starting{}. webAddress={}",
```

Comment:
Should there be a space between `starting` and `{}`?

Reply:
Looking at the usage, the messages prepend a space. (and the empty message can end with a period.